edx_proctoring/templates/proctored_exam/pending-prerequisites.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+<div class="warning sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      This exam is proctored.
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+    This exam is proctored but you have not completed all of the prerequisites for this exam. Either you have not yet attempted these prerequisites or the results of your attempts are still pending. All requirements for this exam must be passed before you can proceed to take this proctored exam. You can see the requirements listed (in order) on your <a href="{{progress_page_url}}">Progress</a> page.
```

Comment:
"You have not completed the prerequisites for this exam. All requirements must be satisfied before you can take this proctored exam and be eligible for credit."
"See your Progress page for a list of requirements in the order that they must be completed."